tfhe/src/core_crypto/fft_impl/fft64/crypto/ggsw.rs

```diff
@@ -746,8 +747,15 @@ pub(crate) fn update_with_fmadd(
                         let out = S::c64s_as_mut_simd(output_fourier).0;
                         let lhs = S::c64s_as_simd(ggsw_poly).0;
 
-                        for (out, &lhs, &rhs) in izip!(out, lhs, rhs) {
-                            *out = simd.c64s_mul(lhs, rhs);
+                        let (lhs0, lhs1) = lhs.split_at(len / 2);
+                        let (rhs0, rhs1) = rhs.split_at(len / 2);
+                        let (out0, out1) = out.split_at_mut(len / 2);
+
+                        for ((out0, out1), (lhs0, lhs1), (rhs0, rhs1)) in
+                            izip!(izip!(out0, out1), izip!(lhs0, lhs1), izip!(rhs0, rhs1),)
+                        {
+                            *out0 = simd.c64s_mul(*lhs0, *rhs0);
+                            *out1 = simd.c64s_mul(*lhs1, *rhs1);
```

Comment:
a comment to explain why this can be better with the prefecthers, same below

Reply:
will do